Backend/Backend/Services/PodcastService.cs

```diff
@@ -83,13 +86,55 @@ public class PodcastService : IPodcastService
             }
             else
             {
+
                 var podcastId = Guid.Parse(id);
                 return await _db.Podcasts!.Include(u => u.Cover).Include(u =>u.Episodes).FirstOrDefaultAsync(u => u.Id == podcastId);
         
             }
         }
 
 
+        public async Task<List<GetPodcastResponse>> GetMyPodcast(HttpContext httpContext)
+        {
+           User? user = await  _authService.IdentifyUserAsync(httpContext);
+            if(user != null)
+            {
+                List<Podcast> podcasts = await _db.Podcasts!.Include(u => u.Episodes).Where(u => u.PodcasterId == user.Id).ToListAsync();
+ 
+                List<GetPodcastResponse> response = new List<GetPodcastResponse>();
+                foreach(var podcast in podcasts)
+                {
+                    response.Add(new GetPodcastResponse()
+                    {
+                        Id = podcast.Id,
+                        Name = podcast.Name,
+                        Description = podcast.Description,
+                        CoverId = podcast.CoverId,
+                        Tags = podcast.Tags,
+                        IsExplicit = podcast.IsExplicit,
+                        Type = podcast.Type,
+                        AverageRating = podcast.AverageRating,
+                        TotalRatings = podcast.TotalRatings,
+                        NoOfEpisode = podcast.Episodes.Count,
+
+                    });
+
+                }
+
+                return response;
+                
+      
+
+
+            }
+            else
+            {
+                throw new Exception("Not found");
```

Comment:
would make this exception consistent with the rest of the code we have, so something like this:
![image](https://github.com/awaazo/awaazo/assets/78050750/2ca780dd-94e4-4f3a-bf61-1d5fdd322951)